src/lua-runtime-fastpm.lua

```diff
@@ -33,31 +33,57 @@ function schema.output_redshifts.action(output_redshifts)
+
+schema.declare{name='omega_m',           type='number', required=false, help='Please use Omega_m. This will be removed eventually.'}
+schema.declare{name='Omega_m',           type='number', required=false, help='Total matter (cdm + baryon + ncdm) density parameter at z=0'}
+schema.declare{name='N_eff',             type='number', required=false, default=3.046}
+schema.declare{name='N_nu',              type='number', required=false, default=0, help="Total number of neutrinos, massive and massless."}
+schema.declare{name='m_ncdm',            type='array:number', required=false, default={}, help="Mass of ncdm particles in eV. Enter in descending order."}
+schema.declare{name='T_cmb',             type='number', required=false, default=0, help="CMB temperature in K, 0 to turn off radiation."}
 schema.declare{name='h',                 type='number', required=true, default=0.7, help="Dimensionless Hubble parameter"}
-schema.declare{name='pm_nc_factor',      type='array:number',  required=true,
-    help="A list of {a, PM resolution}. Add an entry {0.0, 1} to set lpt linear field resolution to 1."}
+schema.declare{name='pm_nc_factor',      type='array:number',  required=true, help="A list of {a, PM resolution}, "}
 schema.declare{name='np_alloc_factor',   type='number', required=true, help="Over allocation factor for load imbalance" }
 schema.declare{name='compute_potential', type='boolean', required=false, default=false, help="Calculate the gravitional potential."}
-schema.declare{name='m_ncdm',            type='array:number', required=false, default={}, help="Mass of ncdm particles. Enter in descending order."}
-schema.declare{name='every_ncdm',        type='number', required=false, default=4, help="Subsample ncdm from cdm every..."}
 schema.declare{name='n_shell',           type='number', required=false, default=10, help="Number of shells of FD distribution for ncdm splitting."}
-schema.declare{name='n_side',            type='number', required=false, default=2, help="Number of sides in HEALPix or fibonacci split. Note that this means n_fib for fibonacci."}
 schema.declare{name='lvk',               type='boolean', required=false, default=false, help="Use the low velocity kernel when splitting FD for ncdm."}
+schema.declare{name='n_side',            type='number', required=false, default=2, help="Number of sides in HEALPix split."}
+schema.declare{name='every_ncdm',        type='number', required=false, default=4, help="Subsample ncdm from cdm every..."}
 schema.declare{name='ncdm_sphere_scheme',type='enum', required=false, default="fibonacci", help="Split sphere with 'fibonacci' or 'healpix'?"}
 schema.ncdm_sphere_scheme.choices = {
     healpix = 'FASTPM_NCDM_SPHERE_HEALPIX',
     fibonacci = 'FASTPM_NCDM_SPHERE_FIBONACCI',
 }
+-- allow backward compatibility wth lowercase o
+function schema.omega_m.action (value)
+     if value ~= nil then
```

Comment:
What about print a message here to warn that lower case omega_m is deprecated and shall use Omega_m instead?
I think print() would do.

Reply:
print or error? print would get lost in the output log file, but an error would force users to change omega_m to Omega_m. Which would you prefer?